Update to add zkSync network from Chainlist.org on "Add zkSync to MetaMask" (#931)
matter-labs/zksync-web-era-docs: docs/build/quick-start/add-zksync-to-metamask.md

Click on "Save" to add the zkSync network to MetaMask.

> ## Import from Chainlist

Reviewer: nit: capitalization
```suggestion
## Import from ChainList
```

> To add zkSync as a custom network in Metamask from Chainlist.org, follow these steps:

Reviewer: nit: Please properly capitalize company names
```suggestion
To add zkSync as a custom network in MetaMask from ChainList, follow these steps:
```

> - Visit https://chainlist.org/chain/324 (Mainnet) or https://chainlist.org/chain/300 (Testnet) on your browser

Reviewer: Please specify Sepolia Testnet
```suggestion
- Visit https://chainlist.org/chain/324 (Mainnet) or https://chainlist.org/chain/300 (Sepolia Testnet)
```

> - Click Connect

Reviewer: Update to exact verbiage
```suggestion
- Click `Connect Wallet`
```

> - Unlock your metamask

Reviewer: Can you clarify what you mean by "Unlock"? I think we can be more precise with a statement of `Connect your MetaMask wallet`, thoughts?

> - Show metamask popup `Allow this site to add a network?` then click Approve.

Reviewer: For instructions, we should write them from the perspective of the user. With that in mind, thoughts about the following edit?

Note: Also remove the ending period, `.`, for consistency
```suggestion
- When prompted with `Allow this site to add a network?`, click `Approve`
```

> - Show `Allow this site to switch the network?` and click Switch

Reviewer: Similar feedback as line prior
```suggestion
- When prompted with `Allow this site to switch the network?`, click `Switch`
```

> Now you should see the zkSync network listed in the Networks section of MetaMask. You can switch to the zkSync network to interact with zkSync applications and contracts.

Reviewer: Should we move this line to `Line 39`?
